src/http-client/helpers.test.ts

```diff
@@ -26,46 +26,44 @@ import {
 describe('Test HttpClient utils', () => {
   it('build auth by tree req options', () => {
     const txnAuthObj = authzTreeJSON;
-    // eslint-disable-next-line max-len, prettier/prettier
-    const expectedUrl = 'https://openam.example.com/am/json/realms/root/authenticate?authIndexType=composite_advice&authIndexValue=%3CAdvices%3E%3CAttributeValuePair%3E%3CAttribute%20name%3D%22AuthenticateToServiceConditionAdvice%22%2F%3E%3CValue%3Eabc%3C%2FValue%3E%3C%2FAttributeValuePair%3E%3C%2FAdvices%3E';
+    const expectedUrl =
+      // eslint-disable-next-line max-len, prettier/prettier
+      'https://openam.example.com/am/json/realms/root/authenticate?authIndexType=composite_advice&authIndexValue=%3CAdvices%3E%3CAttributeValuePair%3E%3CAttribute%20name%3D%22AuthenticateToServiceConditionAdvice%22%2F%3E%3CValue%3Eabc%3C%2FValue%3E%3C%2FAttributeValuePair%3E%3C%2FAdvices%3E';
 
     const output = buildAuthzOptions(txnAuthObj, 'https://openam.example.com/am/', 0);
     expect(output.url).toStrictEqual(expectedUrl);
   });
 
   it('build auth by txn req options', () => {
     const txnAuthObj = authzTxnJSON;
-    // eslint-disable-next-line max-len, prettier/prettier
-    const expectedUrl = 'https://openam.example.com/am/json/realms/root/authenticate?authIndexType=composite_advice&authIndexValue=%3CAdvices%3E%3CAttributeValuePair%3E%3CAttribute%20name%3D%22TransactionConditionAdvice%22%2F%3E%3CValue%3Eabc%3C%2FValue%3E%3C%2FAttributeValuePair%3E%3C%2FAdvices%3E';
+    const expectedUrl =
+      // eslint-disable-next-line max-len, prettier/prettier
+      'https://openam.example.com/am/json/realms/root/authenticate?authIndexType=composite_advice&authIndexValue=%3CAdvices%3E%3CAttributeValuePair%3E%3CAttribute%20name%3D%22TransactionConditionAdvice%22%2F%3E%3CValue%3Eabc%3C%2FValue%3E%3C%2FAttributeValuePair%3E%3C%2FAdvices%3E';
     const output = buildAuthzOptions(txnAuthObj, 'https://openam.example.com/am/', 0);
     expect(output.url).toStrictEqual(expectedUrl);
   });
 
-  it('examines response for IG auth by tree', async (done) => {
```

Comment:
As an upgrade to Jest 27, it is illegal to use both, done and async (returning a promise). I removed `done` from the tests since we implicitly return a promise with `async` `await`

Reply:
This is cool. Back in the day, Jest didn't want to cleanly exit when tests would finish, so I dropped `done()` within them, and it fixed it 🤷  If they now cleanly exit without it, I'm good!